src/_guides/language/effective-dart/design.md

```diff
@@ -1555,14 +1555,19 @@ bool convertToBool(dynamic arg) {
 ### DO use `Future<void>` as the return type of asynchronous members that do not produce values.
 
 When you have a synchronous function that doesn't return a value, you use `void`
-as the return type. The asynchronous equivalent is `Future<void>`.
+as the return type. The asynchronous equivalent for a method which does not
+produce a value, but should have the work awaited, is `Future<void>`.
```

Comment:
Is "should" required here? Mightn't it be optional? If so, maybe:
...but that the caller might need to await, ...

Reply:
Yeah it is optional. Will take your suggested wording.